src/AutoIndexer.ts

```diff
@@ -6,6 +6,22 @@ import ucFirst from '@/ucFirst';
 import removeFileExtension from '@/removeFileExtension';
 import { GetIndexYamlOptions } from '@/interfaces/GetIndexYamlOptions';
 
+function getMethodFromFileName(fileName: string): string {
+  return fileName.split('.')[0];
+}
+
+export function buildIndexFromPath(cleanPath: string, trimOpts?: any) {
+  const dir = path.dirname(cleanPath);
```

Comment:
This is re-doing some calculations that are already done if you're using it in the `AutoIndexer`.

Could split it into functions, one for external consumption that calculates filename/dir etc... and one for use inside the `AutoIndexer`

Will do that if you're happy with the approach in general.

Reply:
@johndcarmichael thoughts?

Reply:
Apologies, pretty slammed at the moment. Will take a look this evening.